Library/Formula/rack.rb

```diff
@@ -0,0 +1,20 @@
+class Rack < Formula
+  desc "A CLI for Rackspace"
+  homepage "https://github.com/rackspace/rack"
+  url "https://github.com/rackspace/rack/archive/1.0.0-beta.1.tar.gz"
+  version "1.0.0-beta.1"
+  sha256 "2ab9513fee4a23667e89c05548be26b4f15a387a8be1e1896936f6e253e05c51"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "go", "get", "-d"
```

Comment:
Looks like this issue was fixed by a commit since the release: https://github.com/rackspace/rack/commit/833593b72c6e8ad0577844d480d75aa924744e25